cpanspec

```diff
@@ -1895,6 +1681,211 @@ sub decode_latin_or_utf8 {
     return $string;
 }
 
+sub readmetajson {
```

Comment:
```suggestion
sub read_meta_json {
```
Minor point, but imho this is more readable with underscores

Reply:
done